symengine/eval_mpfr.cpp

```diff
@@ -213,15 +213,20 @@ class EvalMPFRVisitor : public BaseVisitor<EvalMPFRVisitor> {
     };
 
     void bvisit(const Constant &x) {
+
+        mpfr_t one_;
+        mpfr_init2(one_,mpfr_get_prec(result_));
+        mpfr_set_si(one_,1,rnd_);
         if (x.__eq__(*pi)) {
             mpfr_const_pi(result_, rnd_);
         } else if (x.__eq__(*E)) {
-            mpfr_const_euler(result_, rnd_);
+            mpfr_exp(result_,one_, rnd_);
         } else if (x.__eq__(*EulerGamma)) {
-            mpfr_const_euler(result_, rnd_);
+            mpfr_exp(result_,one_, rnd_);
         } else {
             throw std::runtime_error("Constant " + x.get_name() + " is not implemented.");
```

Comment:
Though this is not causing the error I have cleared it after line 223 in the `else if`.